### PR TITLE
feat: redirect chanage property reactions

### PR DESCRIPTION
**Pull request by Blackheart340**

### CODE REVIEW

**Comment by TimeRaider** [comment]
Can you elaborate on why you need this?

**Comment by Blackheart340** [comment]
@TimeRaider 'case sometimes you have a case, when your "redirect" property generated dynamic.
For example: request to API -> response -> you're making a decision where you redirect

**Comment by TimeRaider** [comment]
@Blackheart340 Wouldn't just using an action creator be easier?

File: src/components/Fragment.js [hunk 1]
```diff
@@ -17,20 +17,32 @@ class Fragment extends BaseRouterComponent {
         };
     }
 
+    componentWillReceiveProps(newProps) {
+
+        const { redirect } = this.props;
+        const newRedirect = newProps.redirect;
```
Comment (Blackheart340) [inline src/components/Fragment.js]: 'case newRedirect will be `undefined`.
We can make like that, for example:
```js
const { redirect as newRedirect }  = newProps;
```
But I'm not sure what it'll be better.

Comment (TimeRaider) [inline src/components/Fragment.js]: alright, i guess it's fine :)

**Comment by Blackheart340** [comment]
@TimeRaider can you publish a package with those changes?

**Pull request merged**